_config.yml

```diff
@@ -34,6 +34,7 @@ default_js:
   - "https://cdn.jsdelivr.net/docsearch.js/2/docsearch.min.js"
   - "/scripts/search.js"
   - "/scripts/clipboard.js"
+  - "/scripts/open-in-repl.js"
```

Comment:
I don't know, I like the separation of responsibility here.

We do update the same JavaScript code blocks twice but I think it's not a big deal.